### PR TITLE
[ZT] Area 1 + Access configuration

### PR DESCRIPTION
**Pull request by ranbel**

### CODE REVIEW

**Comment by ranbel** [comment]
Preview links for updated pages:
https://ranbel-area1-access.cloudflare-docs-7ou.pages.dev/cloudflare-one/tutorials/area-1/
https://ranbel-area1-access.cloudflare-docs-7ou.pages.dev/cloudflare-one/applications/configure-apps/saas-apps/
https://ranbel-area1-access.cloudflare-docs-7ou.pages.dev/cloudflare-one/applications/configure-apps/

File: content/cloudflare-one/applications/configure-apps/saas-apps/area-1.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+1. In the [Zero Trust dashboard](https://dash.teams.cloudflare.com), go to **Access** > **Applications**.
+
+2. Select **Add an application**.
+
+3. Select **SaaS**.
+
+4. In the **Application** field, enter `Area 1` and select **Area 1**. (Area 1 is not currently listed in the default drop-down menu.)
+
+5. Enter the following values for your application configuration:
+    - **Entity ID**: `https://horizon.area1security.com`
```
Comment (kennyj42) [inline content/cloudflare-one/applications/configure-apps/saas-apps/area-1.md]: @ranbel is "horizon" an account specific tag vs. something that spans all accounts?

Comment (ranbel) [reply content/cloudflare-one/applications/configure-apps/saas-apps/area-1.md]: Great question. I will have to check with Todd Murray.

Comment (ranbel) [reply content/cloudflare-one/applications/configure-apps/saas-apps/area-1.md]: From Todd: "It’s a single front end portal for all customers."

File: content/cloudflare-one/applications/configure-apps/saas-apps/area-1.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+3. (Optional) To require users to sign in through Access, set **SSO Enforcement** to _All_.
+
+4. In **SAML SSO Domain**, enter `<your-team-name>.cloudflareaccess.com`.
+
+5. Get your Metadata XML file:
+
+    1. In the Zero Trust dashboard, copy the **SSO Endpoint** for your application. The URL is of the form: `https://<your-team-name>.cloudflareaccess.com/cdn-cgi/access/sso/saml/<app-id>`.
+
+    2. In a new browser tab, paste the **SSO Endpoint** and append `/saml-metadata` to the end of the URL. For example,
```
Comment (ranbel) [inline content/cloudflare-one/applications/configure-apps/saas-apps/area-1.md]: @kennyj42 @jroyal According to Todd, `https://<your-team-name>.cloudflareaccess.com/cdn-cgi/access/sso/saml/<app-id>/saml-metadata` is changing over time. This means the Metadata XML setting in Area 1 becomes outdated and the integration breaks. Is the Access saml-metadata supposed to change?

Comment (jroyal) [reply content/cloudflare-one/applications/configure-apps/saas-apps/area-1.md]: Changing how? I don't think anything in except maybe keys should change.

**Review by dcpena — APPROVED** [review]

**Pull request merged**